libyul/backends/evm/EVMCodeTransform.h

```diff
@@ -57,6 +59,20 @@ struct CodeTransformContext
 	std::map<Scope::Function const*, AbstractAssembly::LabelID> functionEntryIDs;
 	std::map<Scope::Variable const*, int> variableStackHeights;
 	std::map<Scope::Variable const*, unsigned> variableReferences;
+
+	struct JumpInfo
+	{
+		AbstractAssembly::LabelID label;  ///< Jump's LabelID to jump to.
+		int stackHeightAdjust;            ///< Stack height after the jump.
```

Comment:
It's not the adjustment, it is the "absolute" height, isn't it?

Reply:
actually it is the absolute value, yes. (wasn't sure about that fact in the beginning).